contract_manager/package.json

```diff
@@ -21,6 +21,7 @@
     "url": "git+https://github.com/pyth-network/pyth-crosschain.git"
   },
   "dependencies": {
+    "@coral-xyz/anchor": "^0.28.1-beta.1",
```

Comment:
this package was using `@coral-xyz/anchor` but didnt pin the version here hence it was using an old version of the package that some other packages are using, generally I think its a good idea to pin versions specific to packages